fyi.zoey.Boop-GTK.json

```diff
@@ -1,7 +1,7 @@
 {
     "app-id": "fyi.zoey.Boop-GTK",
     "runtime": "org.freedesktop.Platform",
-    "runtime-version": "20.08",
```

Comment:
any reason for not using 22.08 here?

Reply:
Steady wins the race? I prefer incremental updates over rushing things out of the door. Perhaps 22.08 also works, but let's not be too hasty.